setup.cfg

```diff
@@ -0,0 +1,72 @@
+[metadata]
+name = scikit-aero
+version = 0.2.dev0
```

Comment:
There's a way to use the variable from `src/skaero/version.py` https://setuptools.readthedocs.io/en/latest/setuptools.html?highlight=version#configuring-setup-using-setup-cfg-files